server/statshouse/statshouse-manager.h

```diff
@@ -66,9 +77,15 @@ class StatsHouseManager : vk::not_copyable {
    */
   void add_init_master_stats(uint64_t total_init_ns, uint64_t confdata_init_ns);
 
+  /**
+   * before calling the method, be sure to is_extended_instance_cache_stats_enabled() is true
+   */
+  void add_extended_instance_cache_stats(const std::string_view type, const std::string_view status, const string &key, uint64_t size = 0);
```

Comment:
`std::string_view` is always const
```suggestion
  void add_extended_instance_cache_stats(std::string_view type, std::string_view status, const string &key, uint64_t size = 0);
```